Dockerfile

```diff
@@ -3,13 +3,18 @@ FROM alpine:3.4
 ENV LANG=C.UTF-8 \
     JAVA_HOME=/usr/lib/jvm/default-jvm/jre \
     PATH=${PATH}:${JAVA_HOME}/bin \
-    CHE_HOME=/home/user/che
+    CHE_HOME=/home/user/che \
+    DOCKER_VERSION=1.6.0 \
+    DOCKER_BUCKET=get.docker.com
 
 RUN echo "http://dl-4.alpinelinux.org/alpine/edge/community" >> /etc/apk/repositories && \
     apk upgrade --update && \
-    apk add --update docker openjdk8 sudo bash && \
+    apk add --update ca-certificates curl openssl openjdk8 sudo bash && \
+    curl -sL "https://${DOCKER_BUCKET}/builds/Linux/x86_64/docker-$DOCKER_VERSION" \
+    > /usr/bin/docker; chmod +x /usr/bin/docker && \
```

Comment:
I think that **> /usr/bin/docker** is an ending of previous command should not be moved to a new line.
Also **; chmod +x /usr/bin/docker** means skip error of previous step. It is better to use **&&** instead